media/css/m24/components/footer-refresh.scss

```diff
@@ -0,0 +1,675 @@
+    @media(min-width: $screen-md) {
+        width: 20%;
+    }
+
+    &.moz24-links-section {
+        display: grid;
+        grid-template-rows: 1fr 1fr;
+        grid-template-columns: 1fr 1fr;
+        place-items: stretch start;
+
+        @media(min-width: $screen-md) {
+            width: 70%;
+            display: flex;
+            flex-direction: row;
+            justify-content: space-between;
+            align-items: start;
+        }
+    }
+}
+
+.moz24-footer-refresh-social-wrapper {
+    display: flex;
+    align-items: center;
+    margin-bottom: $spacer-lg;
+
+    @media(min-width: $screen-md) {
+        flex-direction: column;
+        justify-content: center;
+        align-items: flex-start;
+    }
+}
+
+
+.moz24-footer-heading-social {
+    padding: 0;
+    font-family: $primary-font;
+    color: $color-white;
+    margin: 0;
+    font-size: $text-title-sm;
+
+    @media(min-width: $screen-md) {
+        margin-bottom: $spacer-sm;
+    }
+}
+
+.moz24-footer-links-social {
+    margin: 0 0 0 $spacer-lg;
+    display: flex;
+
+    @media(min-width: $screen-md) {
+        margin: 0;
+    }
+
+    li {
+        margin-right: $spacer-2xs;
+        display: inline-block;
+
+        a {
+            background-repeat: no-repeat;
+            border-bottom: 1px solid rgba(0,0,0,0);
+            direction: ltr;
+            display: block;
+            height: 16px;
+            overflow: hidden;
+            text-align: left;
+            text-indent: -9999px;
+            white-space: nowrap;
+            width: 16px;
+            @include bidi(((margin-right, 16px, 0), (margin-left, 0, 16px)));
+
+            &:hover,
+            &:focus,
+            &:active {
+                outline: 1px dotted $color-white;
+                outline-offset: $spacing-xs;
+                border-bottom-color: transparent;
+                background-repeat: no-repeat;
+            }
+
+            &.linkedin {
+                background-image: url('/media/protocol/img/icons/social/linkedin/white.svg');
+            }
+
+            &.tiktok {
+                background-image: url('/media/protocol/img/icons/social/tiktok/white.svg');
+            }
+
+            &.spotify {
+                background-image: url('/media/protocol/img/icons/social/spotify/white.svg');
+            }
+
+            &.mastodon {
+                background-image: url('/media/protocol/img/icons/social/mastodon/white.svg');
+                background-size: contain;
+            }
+
+            &.twitter {
+                background-image: url('/media/protocol/img/icons/social/x/white.svg');
+            }
+
+            &.instagram {
+                background-image: url('/media/protocol/img/icons/social/instagram/white.svg');
+            }
+
+            &.youtube {
+                background-image: url('/media/protocol/img/icons/social/youtube/white.svg');
+            }
+        }
+    }
+}
+
+.moz24-footer-label {
+    color: $token-color-medium-gray;
```

Comment:
Updated the color to be white and bold. It is not super distinguishable with the links below, but I am hesitate to add the underline, in case they will be misunderstood as links. Another option would be to update the headings to use the Semi-slab font
<img width="673" alt="Screenshot 2024-09-06 at 12 56 01 PM" src="https://github.com/user-attachments/assets/19f5eccf-09e7-4892-b219-860c1ea49b43">

Reply:
<img width="670" alt="Screenshot 2024-09-06 at 12 59 37 PM" src="https://github.com/user-attachments/assets/671498be-9498-466b-8b15-a3acaded9fd9">